### ISSUE TITLE
Invalid `-ftabstop` values print wrong diagnostic

### ISSUE THREAD

**Issue by tbaederr** [llvm/llvm-project#62912]
See: https://godbolt.org/z/463j379xo

```console
warning: ignoring invalid -ftabstop value '8', using default value 8
```

**Comment by csmoe** [comment]
I'll take a look as my first contribution to llvm and learn C++.

**Comment by huangperry** [comment]
Hi, I'm new to LLVM and interested in tackling this issue!

**Comment by csmoe** [comment]
I submitted a patch days ago, https://reviews.llvm.org/D151429
